SimGeneral/MixingModule/python/caloTruthProducer_cfi.py

```diff
@@ -12,17 +12,17 @@
 	maximumPreviousBunchCrossing = cms.uint32(0),
 	maximumSubsequentBunchCrossing = cms.uint32(0),
 	simHitCollections = cms.PSet(
-            hgc = cms.VInputTag(
-                cms.InputTag('g4SimHits','HGCHitsEE'),
-                cms.InputTag('g4SimHits','HGCHitsHEfront'),
-                cms.InputTag('g4SimHits','HcalHits')
-            ),
+#            hgc = cms.VInputTag(
+#                cms.InputTag('g4SimHits','HGCHitsEE'),
+#                cms.InputTag('g4SimHits','HGCHitsHEfront'),
+#                cms.InputTag('g4SimHits','HcalHits')
+#            ),
```

Comment:
Why commenting the HGC ? 
can be made era customizable Run3 (ECAL+HCAL) vs phase2 ?

Reply:
Our plan was to just make a dedicated CMSSW release with these changes, but if we wish we can customize it with some boolean. How do you prefer to proceed?

Reply:
New era added Era_Run3_ecalClustering